app/src/main/java/com/cooper/wheellog/views/WheelView.kt

```diff
@@ -140,13 +142,26 @@ class WheelView(context: Context, attrs: AttributeSet?) : View(context, attrs) {
                         String.format(Locale.US, "%.0f " + resources.getString(R.string.km), mTotalDistance)
                     }
                 },
+
+                ViewBlockInfo(resources.getString(R.string.battery)) { String.format(Locale.US, "%d %%", mBattery) },
+
                 ViewBlockInfo(resources.getString(R.string.current)) { String.format(Locale.US, "%.2f " + resources.getString(R.string.amp), mCurrent) },
+
+                ViewBlockInfo(resources.getString(R.string.maxcurrent)) { String.format(Locale.US, "%.0f " + resources.getString(R.string.amp), WheelData.getInstance().getMaxCurrent()) },
+
                 ViewBlockInfo(resources.getString(R.string.power),
                         { String.format(Locale.US, "%.2f " + resources.getString(R.string.watt), WheelData.getInstance().powerDouble) }, false),
+
+                ViewBlockInfo(resources.getString(R.string.maxpower),
+                        { String.format(Locale.US, "%.0f " + resources.getString(R.string.watt), WheelData.getInstance().getMaxPower()) }, false),
```

Comment:
может лучше показывать с 2 знаками после запятой как обычная мощность?
```suggestion
                        { String.format(Locale.US, "%.2f " + resources.getString(R.string.watt), WheelData.getInstance().getMaxPower()) }, false),
```

Reply:
я против, нет смысла смотреть максимальную мощность в виде 4567,77 вт